vnext/Scripts/Microsoft.ReactNative.Cxx.nuspec

```diff
@@ -21,6 +21,8 @@
   </metadata>
```

Comment:
100% agree, they belong next to the projects, I have held off until we have signing enabled since I don't want to hinder Jon's work to get signing up and running.